Source/Csla/Configuration/ConfigurationExtensions.cs

```diff
@@ -54,13 +54,13 @@ public static IServiceCollection AddCsla(this IServiceCollection services, Actio
       services.TryAddTransient((p) => new Channels.Local.LocalProxyOptions());
       services.AddTransient<Channels.Local.LocalProxy, Channels.Local.LocalProxy>();
       // Add server-side data portal; this is always necessary to support RunLocal
-      cslaOptions.DataPortal().ApplyConfiguration();
+      cslaOptions.DataPortal(options => options.ApplyConfiguration());
```

Comment:
Hmm, yeah, that looks a bit unusual, doesn't it?

Can we do something like this instead - solely from _inside_ of Csla, never from outside:

```csharp
cslaOptions.DataPortalClientOptions.ApplyConfiguration();
```

Reply:
`ApplyConfiguration` is not public, so yes, we should be able to do that.